Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/COREX12BatchSubmitRequest.properties

```diff
@@ -51,12 +51,12 @@ ActiveParticipant_Dest.RoleIDCode.@code=110152
 ActiveParticipant_Dest.RoleIDCode.@codeSystemName=DCM
 ActiveParticipant_Dest.RoleIDCode.@displayName=Destination
 
-AuditSourceIdentification.@AuditEnterpriseSiteID=Gateway 1
+AuditSourceIdentification.@AuditEnterpriseSiteID=DoD
```

Comment:
We can have at project level, But I would like to handle this as separate issue. I can create ticket in backlog.